packages/react-ui/src/app/features/builder/builder-hooks.ts

```diff
@@ -619,19 +630,13 @@ const updateFlowVersion = (
     deleteChatRequest(state.flowVersion, operation.request.name);
   }
 
-  async function deleteChatRequest(flowVersion: FlowVersion, stepName: string) {
-    try {
-      const stepDetails = flowHelper.getStep(flowVersion, stepName);
-      const blockName = stepDetails?.settings?.blockName;
-      const chat = await aiChatApi.open(
-        newFlowVersion.flowId,
-        blockName,
-        stepName,
-      );
-      await aiChatApi.delete(chat.chatId);
-    } catch (err) {
-      console.error(err);
-    }
+  if (operation.type === FlowOperationType.DUPLICATE_ACTION) {
```

Comment:
@alexandrudanpop did you see my comment on the ticket, that it might be better to do this on move, rather on duplicate?

Reply:
will do a separate PR for drag as discussed and will keep the new behavior for duplicate